charts/k8s-image-swapper/README.md

```diff
@@ -10,7 +10,8 @@ Mirror images into your own registry and swap image references automatically.
 
 | Name | Email | Url |
 | ---- | ------ | --- |
-| estahn | enrico.stahn@gmail.com |  |
+| estahn | <enrico.stahn@gmail.com> |  |
+| tcyran | <tcyran@gmail.com> |  |
```

Comment:
@tcyran Typically there is a distinction between `maintainers` and `contributors`. This table is showing only maintainers, not contributors. Could you remove your name here? You will be mentioned in the release notes.

Reply:
Clear, removed